release-kong.sh

```diff
@@ -108,9 +108,17 @@ function push_package() {
 
   set -x
 
-  local release_args="--package-type gateway --publish"
+  local release_args="--package-type gateway"
   if [[ "$EDITION" == "enterprise" ]]; then
     release_args="$release_args --enterprise"
+    # enterprise pre-releases go to `/internal/`
+    if [[ "$OFFICIAL_RELEASE" == "false" ]]; then
+      release_args="$release_args --internal"
+    else
+      release_args="$release_args --publish"
+    fi
```

Comment:
In languages other than bash, the general wisdom is you should always replace `if not x ... else ...` with `if x ... else ...`.

```suggestion
    if [[ "$OFFICIAL_RELEASE" == "true" ]]; then
      release_args="$release_args --publish"
    else
      release_args="$release_args --internal"
    fi
```